README.md

```diff
@@ -0,0 +1,62 @@
+# Picarro Data Upload and Download
+This github repository contains all scripts for uploading and downloading data from Google Cloud Storage and Firestore
+
+* The firestore database consists of *hourly aggregate* data for the Picarro machine,combined into `Year/Month` collections
+* Google Cloud Storage holds the raw data with no aggregation
+
+## Setting Up Your Machine
+To run these scripts, you'll need the following:
+* Python version 3.5 or greater. This may be pre-installed on your system; to check use `python3 --version`
```

Comment:
We shouldn't support lower version of Python now. Your pyproject.toml actually stipulates Python 3.10+.

The executable of Python might not always be `python3`. It could also be `python`. Let's make sure that we add `python` as well